### PR TITLE
[T4A6][W14-B4]Abel Lim

### PR DESCRIPTION
**Pull request by abellimz**

### CODE REVIEW

**Comment by abellimz** [comment]
Ready for review

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class UtilsTest {
+	
+	@Test 
+	public void isAnyNull_allNull_returnTrue(){
```
Comment (damithc) [inline test/java/seedu/addressbook/common/UtilsTest.java:13]: Good job following the naming convention :+1:

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+	    ArrayList<Object> testList = new ArrayList<>();
+	    testList.add(1);
+	    Assert.assertTrue(Utils.elementsAreUnique(testList));
+	}
+	
+	@Test 
+	public void elementsAreUnique_duplicateElements_returnFalse(){
+	    Object object = new Object();
+	    Object[] testArray = new Object[]{object, object};
+	    List<Object> testList = Arrays.asList(testArray);
```
Comment (damithc) [inline test/java/seedu/addressbook/common/UtilsTest.java:35]: You can use `List<Integer> list = Arrays.asList(1, 1, 1, 2)` to create and initialize a list in one line.

**Comment by damithc** [comment]
Good effort @abellimz  
Some comments added. Please close the PR after reading comments.

**Pull request closed without merging**